src/Components/RecsListTable/RecsListTable.js

```diff
@@ -418,12 +418,26 @@ const RecsListTable = ({ query }) => {
     return pruneFilters(localFilters, FILTER_CATEGORIES);
   };
 
+  console.log(RECS_LIST_INITIAL_STATE);
   const activeFiltersConfig = {
+    showDeleteButton: true,
     deleteTitle: intl.formatMessage(messages.resetFilters),
     filters: buildFilterChips(),
     onDelete: (_event, itemsToRemove, isAll) => {
-      if (isAll) {
+      if (
+        itemsToRemove.length === 2 &&
+        itemsToRemove[0].chips[0].name.props.children === '1 or more' &&
+        itemsToRemove[1].chips[0].name.props.children === 'Enabled' &&
+        isAll
+      ) {
```

Comment:
Your solution does work. But:
1) it can be rewritten in a much shorter way if you simply use RECS_LIST_INITIAL_STATE,
2) if the initial filter set for this table will be changed, then you will have to rewrite also this part of the code. This is bad 👎🏼 
3) your solution doesn't care about translations == you have to use intl.translate to fix it, however, this solution becomes even more unnecessary complicated.